### DISCUSSION TITLE
Using AWS_URL in sources

### DISCUSSION THREAD

**Discussion by markdavies** [spatie/statamic-responsive-images#80] [Q&A]
How would I go about getting the value of the environment variable `AWS_URL` appended to all of the source urls? I am use an S3 disk, and in `responsiveImage.blade.php` the value of `$src` appends this, but not within the `$sources` variable. I am trying to serve all assets through Cloudfront.

**Comment by riasvdv** [comment]
I don't think Statamic supports the manipulated images being on S3, could you try generating your own transform with the `{{ glide }}` tag and see if that uses the S3 disk?

**Reply by markdavies** [reply]
Yeah I ended up pointing Cloudfront at it and it works pretty well. I had to customise the view to append the cloudfront cname URL to the front
